handlers/challenge_handler.py

```diff
@@ -115,7 +169,8 @@ def execute(cls, slack_wrapper, args, timestamp, channel_id, user_id, user_is_ad
             raise InvalidCommand("Add CTF failed: Invalid characters for CTF name found.")
 
         # Create the channel
-        response = slack_wrapper.create_channel(name)
+        private_ctf = handler_factory.botserver.get_config_option("private_ctfs")
```

Comment:
Couldn't this be an additional argument for AddCTF command to decide on creation, if the ctf should be created private or open instead of having an option for every ctf?